src/components/ex-live-search-results/ex-live-search-results.scss

```diff
@@ -0,0 +1,101 @@
+.ex-live-search-results {
+  @apply overscroll-contain absolute hidden overflow-y-auto bg-white;
+
+  box-shadow: 0px 3px 45px #0000002c;
+  max-height: calc(100vh - 200px);
+  top: 100%;
+  width: 120%;
+
+  ul.ex-live-search-results__list {
+    @apply ml-0;
+  }
+
+  .ex-search-results-item {
+    @apply block clear-both px-5 pt-4 mt-0 overflow-hidden cursor-pointer;
+
+    &__media {
+      @apply float-right;
+
+      max-width: 150px;
+    }
+
+    &__topic {
+      @apply tracking-default text-xxs flex items-center mt-0 mb-1 font-light leading-tight text-black uppercase;
```

Comment:
For consistency reasons you could add the "arrow" to the live search result item.

![image](https://user-images.githubusercontent.com/88790097/141468582-5f40fed8-e615-4890-8b28-e35315183a8b.png)

Reply:
added the arrow